modules/clientutils.js

```diff
@@ -68,6 +69,54 @@
                     });
         }
 
+        function image_difference(imageData, patternData, coords, delta) {
```

Comment:
if this function is about to return a boolean, how about prefix it by "is"? something like ``is_image_different`` sounds better to me :)